### PR TITLE
licensing: add cody only license plan and hide UI elements

### PR DESCRIPTION
**Pull request by kopancek**
## Description

A quick POC for showing how a licensing split for Enterprise-Cody-Only plan might look and how we might enforce gating features in the UI only.

## Video
https://www.loom.com/share/835fc8acbd684ffeb24ef0c6d71a6364

## Test plan

Tested locally, but it is a draft PR for now...

### CODE REVIEW

File: dev/linters/bodyclose/BUILD.bazel [hunk 1]
```diff
@@ -5,8 +5,5 @@ go_library(
     srcs = ["bodyclose.go"],
     importpath = "github.com/sourcegraph/sourcegraph/dev/linters/bodyclose",
     visibility = ["//visibility:public"],
-    deps = [
```
Comment (burmudar) [inline dev/linters/bodyclose/BUILD.bazel:8]: These change are breaking the pipeline

**Comment by burmudar** [comment]
@erzhtor / @kopancek if you run `bazel configure` it should fix the bazel issues in `dev/linters`

File: dev/linters/bodyclose/BUILD.bazel (hunk 1 above)
Comment (burmudar) [reply dev/linters/bodyclose/BUILD.bazel]: Weird. What's your output of `bazel version` ?

Comment (erzhtor) [reply dev/linters/bodyclose/BUILD.bazel]:
```
Bazelisk version: development
Aspect CLI version: 5.5.4
Starting local Bazel server and connecting to it...
Build label: 6.1.1
Build target: bazel-out/darwin_arm64-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Wed Mar 15 15:45:56 2023 (1678895156)
Build timestamp: 1678895156
Build timestamp as int: 1678895156
```

**Pull request closed without merging**